store/src/java/com/zimbra/cs/service/account/GetInfo.java

```diff
@@ -275,7 +275,11 @@ private static int getProfileId(Mailbox mbox, OperationContext octxt) throws Ser
                 }
             }
         } catch (ServiceException exception) {
-            ZimbraLog.account.error("can't get profile image id", exception);
+            if (MailServiceException.NO_SUCH_FOLDER.equals(exception.getCode())) {
+                ZimbraLog.account.debug("Profile image folder doesn't exist");
+            } else {
+                ZimbraLog.account.error("can't get profile image id", exception);
```

Comment:
Split this 
ZimbraLog.account.error("can't get profile image id:%s", exception.getmessage());
ZimbraLog.account.debug("can't get profile image id", exception.getmessage());